yarn.lock

```diff
@@ -4912,6 +4912,14 @@ graphql-middleware@1.2.6:
   dependencies:
     graphql-tools "^3.0.2"
 
+graphql-parse-resolve-info@^4.12.3:
```

Comment:
this looks like the defacto solution to see if a field exists in a query, i looked in the project and couldn't find any other other helpers so i created one